types/rational.go

```diff
@@ -1,4 +1,4 @@
-package types
+package sdk
```

Comment:
what's the intention of calling the package sdk? that doesn't really make sense to me. I will mention that a standard design pattern I've seen is to have the root directory of the repo act as the types folder. - to me at that point it would make sense to have the package named sdk